addOns/spiderAjax/src/main/java/org/zaproxy/zap/extension/spiderAjax/automation/AjaxSpiderJob.java

```diff
@@ -184,22 +172,11 @@ public void runJob(
         progress.info(
                 Constant.messages.getString(
                         "automation.info.urlsfound", this.getType(), numUrlsFound));
-        if (numUrlsFound < this.failIfFoundUrlsLessThan) {
-            progress.error(
-                    Constant.messages.getString(
-                            "automation.error.urlsfound",
-                            this.getType(),
-                            numUrlsFound,
-                            this.failIfFoundUrlsLessThan));
-        }
-        if (numUrlsFound < this.warnIfFoundUrlsLessThan) {
-            progress.warn(
-                    Constant.messages.getString(
-                            "automation.error.urlsfound",
-                            this.getType(),
-                            numUrlsFound,
-                            this.failIfFoundUrlsLessThan));
-        }
+        Control.getSingleton()
+                .getExtensionLoader()
+                .getExtension(ExtensionStats.class)
+                .getInMemoryStats()
+                .counterInc("spiderAjax.urls.added", numUrlsFound);
```

Comment:
Ideally this would be done in the main ajax spider code so that its incremented as urls are added. Could be a good time to add other stats (like out of scope requests) but could be left for another PR (which someone else could do).